packages/smooth_app/lib/pages/pantry/pantry_page.dart

```diff
@@ -219,17 +194,25 @@ class PantryPage extends StatelessWidget {
               context: context,
             );
           }
-          return Padding(
-            key: ValueKey<String>(product.barcode),
-            padding:
-                const EdgeInsets.symmetric(horizontal: 12.0, vertical: 8.0),
-            child: Card(
-              color: SmoothTheme.getColor(
-                colorScheme,
-                Colors.grey,
-                ColorDestination.SURFACE_BACKGROUND,
+          return Dismissible(
+            background: Container(color: colorScheme.background),
+            key: Key(barcode),
+            onDismissed: (final DismissDirection direction) async {
+              pantry.removeBarcode(barcode);
+              await _save(userPreferences);
+            },
+            child: Padding(
+              key: ValueKey<String>(product.barcode),
+              padding:
+                  const EdgeInsets.symmetric(horizontal: 12.0, vertical: 8.0),
+              child: Card(
```

Comment:
Please use SmoothCard, to keep the app theme

Reply:
Fixed